staging/src/kubevirt.io/api/instancetype/v1beta1/types.go

```diff
@@ -101,6 +101,12 @@ type VirtualMachineInstancetypeSpec struct {
 	// +listType=atomic
 	GPUs []v1.GPU `json:"gpus,omitempty"`
 
+	// Optionally defines any panic devices associated with the instancetype.
+	//
+	// +optional
+	// +listType=atomic
+	Panics []v1.Panic `json:"panics,omitempty"`
```

Comment:
We only use instance types for attributes related to host related resources such as vCPUs, memory, GPUs and HostDevices etc. Everything else that's software defined like vTPMs, EFI etc resides in our preference CRDs. Can you move this over there as a `spec.devices.preferredPanics` attribute?